package-lock.json

```diff
@@ -588,7 +588,7 @@
     },
     "abstract-leveldown": {
       "version": "0.12.4",
-      "resolved": "https://registry.npmjs.org/abstract-leveldown/-/abstract-leveldown-0.12.4.tgz",
+      "resolved": "http://registry.npmjs.org/abstract-leveldown/-/abstract-leveldown-0.12.4.tgz",
```

Comment:
if there are no package.json changes, we shouldn't be updating package-lock.json?